aten/src/ATen/core/ATenDispatch.cpp

```diff
@@ -7,4 +7,32 @@ ATenDispatch & globalATenDispatch() {
   return singleton;
 }
 
+void* ATenOpTable::getFallbackOp(TensorTypeId tid) const {
+  // TODO: an alternate strategy here would be to mask out the dead key
+  // and then redispatch gain (automatic delegation).  I haven't done this
+  // for now to make it easier to smoke out error cases.
```

Comment:
Generic fallbacks are specific to a tensor type. So for example, if I register a "lazy tensor" generic fallback, it will get called if there ever is a lazy tensor *anywhere* in the argument list.

Reply:
I am confused when you guys talk about `Lazy Tensor` ... is it our `Lazy Tensor` or now you have implemented some of your own? :grin:

Reply:
@dlibenzi I'm talking about @bwasti's project :) Although we are keeping XLA in mind as we work on this workpath.